core/coreobjects/include/coreobjects/core_event_args_impl.h

```diff
@@ -235,7 +235,7 @@ inline bool CoreEventArgsImpl::validateParameters() const
         case CoreEventId::DeviceLockStateChanged:
             return parameters.hasKey("IsLocked");
         case CoreEventId::StatusChanged:
-            return parameters.getCount() == 1;
+            return parameters.getCount() == 2;
```

Comment:
I'd delete this check. I'm not exactly sure if this is backwards compatible, as older devices will send out events with parameterCount == 1